setup.py

```diff
@@ -42,10 +45,9 @@
     }
 ) + TESTS_REQUIRE
 
-
 setup(
     name='cylc-rose',
-    version='1.0.0',
+    version=__version__,
```

Comment:
The obvious answer is probably to change to pycodestyle which seems to pass this, even though flake8 doesn't.